packages/cli/src/Server.ts

```diff
@@ -2595,6 +2595,20 @@ class App {
 			);
 		}
 
+		if (process.env.NODE_ENV !== 'production') {
+			ignoredEndpoints.push('dev-node-description-updated');
+			this.app.post(
+				'/dev-node-description-updated',
+				async (req: express.Request, res: express.Response) => {
+					const nodeTypes = NodeTypes();
+					nodeTypes.updateNodeTypeDescription(req.body.type, req.body.description);
+					const pushInstance = Push.getInstance();
+					pushInstance.send('nodeDescriptionUpdated', {});
```

Comment:
1. Suggestion:
```suggestion
					NodeTypes().updateNodeTypeDescription(req.body.type, req.body.description);
					Push.getInstance().pushInstance.send('nodeDescriptionUpdated', {});
```
2. Let's avoid passing in `{}` and make it a default in `send`.